### ISSUE TITLE
splitting ifluatex, ifpdf, ... generic

### ISSUE THREAD

**Issue by kberry** [ho-tex/oberdiek#1]
for some future release, if there is ever time ... it would be nice to split ifluatex from oberdiek. it is a separate package in tex live (horrible special case) because we want ifluatex in collection-basic, but not to pull in all of oberdiek there.  oberdiek is in collection-latex.

seems like the same is true for ifpdf.  i don't know if there are other generic packages (i suspect so); if so, would be nice to split those too, so we can have oberdiek-generic in collection-generic and just the real latex stuff in latex.

anyway ... not urgent at all, just posting here since now there is a place to post it.  maybe someday ...

**Comment by davidcarlisle** [comment]
Yes I agree, I much preferred the ifpdf as a separate package as well.

The only reason I put it back is for the hobsub generation and to be honest
I don't think that's really useful.

Especially tying hobsub to requiring all the source files complicates the
build, we could instead make
it just incorporate extracted files from an already installed tree, then it
could be built separately from the
files it incorporates. Or much simpler and just as useful it could be a
compatibility stub that just does

\RequirePackage{...}

for each of the packages and doesn't copy their contents at all.

One day...

**Comment by kberry** [comment]
Or much simpler and just as useful it could be a
    compatibility stub that just does
    \RequirePackage{...}

Yes, I think that is the least bad of the alternatives, as I wrote some
days ago to Heiko.  

    One day...

Yeah.  Or not :).  -k
